docs/connectors/sinks/postgresql.md

```diff
@@ -0,0 +1,267 @@
+
+## Data type mapping
+
+We currently serialize Feldera SQL view records to JSON before sending them to PostgreSQL.
+This means that the ingestion rules are dictated by [`jsonb_populate_recordset`](https://www.postgresql.org/docs/current/functions-json.html#FUNCTIONS-JSON-PROCESSING-TABLE).
+
+:::info
+The following table lists supported PostgreSQL data types.
+More complex types from PostgreSQL might work but are not yet officially supported and tested.
+Please [let us know](https://github.com/feldera/feldera/issues) if you need support for a specific type.
+:::
+
+| Feldera Type      | PostgreSQL Type                      | Comments                                               |
+|-------------------|--------------------------------------|--------------------------------------------------------|
+| BOOL              | BOOL                                 |                                                        |
+| TINYINT           | SMALLINT, INT, BIGINT                | No direct equivalent but can be stored in wider types. |
+| SMALLINT          | SMALLINT                             |                                                        |
+| INT               | INT                                  |                                                        |
+| BIGINT            | BIGINT                               |                                                        |
+| DECIMAL           | DECIMAL                              |                                                        |
+| REAL              | FLOAT                                |                                                        |
+| DOUBLE            | DOUBLE PRECISION                     |                                                        |
+| VARCHAR           | VARCHAR                              |                                                        |
+| TIME              | TIME                                 |                                                        |
+| DATE              | DATE                                 |                                                        |
+| TIMESTAMP         | TIMESTAMP                            | Feldera TIMESTAMPs do not have timezone information.   |
+| VARIANT           | JSON, JSONB                          |                                                        |
+| UUID              | UUID                                 |                                                        |
+| VARBINARY         | BYTEA                                |                                                        |
+| ARRAY             | ARRAY                                |                                                        |
+| User Defined Type | Equivalent PostgreSQL Composite Type |                                                        |
+| MAP               | JSON, JSONB                          | No equivalent type, store as JSON / JSONB.             |
```

Comment:
I suspect this only works for maps with keys of a string type? If so, please document this.